frontend/src/components/core/Container.tsx

```diff
@@ -10,6 +10,27 @@ const createContainer = (Container: any, props: MyProps) => {
   return <Container>{children}</Container>;
 };
 
+export const FlexContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  max-height: 100vh;
+  overflow: hidden;
+`;
+
+export const FlexInfoBar = styled.div`
+  flex: 1 0 100%;
+  padding: 0.5rem;
+  height: 15px;
+  line-height: 15px;
+  text-align: center;
+`;
+
+export const FlexPanel = styled.div`
+  flex: 1;
+  padding: 1rem;
+  background-color: #e3e3e3;
```

Comment:
Minor nit: Move color to Theme

Reply:
Bump @helenli8181

Reply:
I changed the game page to use the Theme background color so it matches our other pages now. The only drawback of this is the header and problem panel are now the same color.